specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/managedClusters.json

```diff
@@ -3794,6 +3794,10 @@
         "securityProfile": {
```

Comment:
Your change should go to the file for 2023-**09**-02-preview instead of 2023-**07**-02-preview

specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview